modules/add_show_availability.R

```diff
@@ -0,0 +1,176 @@
+###############################################
+### Show available data ###
+###############################################
+
+# This is a module to show a short overview of available data
+######################################################################
+# Output Module
+######################################################################
+
+show_availability_output <- function(id) {
+
+  ns <- NS(id)
+  tagList(
+    tableOutput(ns("show_stations")),
+    textOutput(ns("show_stations_text")),
+    dataTableOutput(ns("show_available_data_table")),
+    plotOutput(ns("show_available_data"))
+  )
+}
+
+######################################################################
+# Server Module
+######################################################################
+
+show_availability_server <- function(id,
+                                     data_to_show,
+                                     data_stations,
+                                     time_period
+) {
+
+  moduleServer(id, function(input, output, session) {
+
+    ns <- session$ns
```

Comment:
Can these rows be deleted?

Reply:
Good point, I think they can indeed removed.

Reply:
Yes those can be deleted